packages/client/src/components/SignIn/index.js

```diff
@@ -0,0 +1,186 @@
+    text-align: right;
+    font-size: 13px;
+    padding-top: 10px;
+    color: #7f7d7d;
+    margin: 0;
+    
+    a {
+        color: #00568C;
+    }
+`
+const Title = styled.h1`
+    text-align: center;
+`
+
+const handleMouseDownPassword = (event) => {
+    event.preventDefault();
+};
+
+const Signin = () => {
+    const [counter, setCounter] = useState(0);
+    const { signin } = useAuth();
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    let error = false;
+    const [showPassword, setShowPassword] = useState(false);
+    const [serverError, setServerError] = useState(false);
+    const [remember, setRemember] = useState(false);
+
+    const handleChange = (prop) => (event) => {
+        if (prop === 'email') {
+            setEmail(event.target.value);
+        } else if (prop === 'password') {
+            setPassword(event.target.value);
+        } else if (prop === 'serverError') {
+            setServerError(!serverError);
+        } else if (prop === 'remember') {
+            setRemember(!remember);
+        } else if (prop === 'error') {
+            error = !error;
+        }
+    }
+    const handleClickShowPassword = () => {
+        setShowPassword(!showPassword);
+    };
+    const checkValidation = () => {
+        error = !(email.length > 0 && email.includes('@') && email.includes('.') && password.length > 5);
+    }
+
+    const ShowNotifications = () => {
+        if (counter > 0) {
+            checkValidation();
+        }
+        if (error) {
+            return (
+                <Notification id="sign-in" />
+            )
+        }
+        if (serverError) {
+            return (
+                <Notification id="auth" />
+            )
+        }
+        return null;
+    }
+
+    // noinspection HtmlUnknownTarget
+    return <AuthWrapper>
+        <AuthInner>
+            <form>
+                <Title>Sign In</Title>
+
+                <div className="form-group">
+                    <Text
+                        autoFocus = {true}
+                        id = 'email'
+                        placeholder = "Email"
+                        value = {email}
+                        onChange = {handleChange('email')}
+                        required = {true}
+                    />
+                </div>
+                <div className="form-group">
+                    <Text
+                        id = "password"
+                        placeholder = "Password"
+                        type = {showPassword ? 'text' : 'password'}
+                        value = {password}
+                        onChange = {handleChange('password')}
+                        required = {true}
+                        endAdornment = {{
+                            endAdornment:
+                                <InputAdornment position="end">
+                                    <IconButton
+                                        onClick={() => handleClickShowPassword}
```

Comment:
this should either be `onClick={() => handleClickShowPassword()}` or `onClick={handleClickShowPassword}`